contribs/gmf/src/editing/editFeatureComponent.js

```diff
@@ -541,6 +563,20 @@ Controller.prototype.$onInit = function () {
   if (!this.map) {
     throw new Error('Missing map');
   }
+
+  /**
+   * @type {import("ol/layer/Vector.js").default}
+   */
+  this.highlightVectorLayer = new olLayerVector({
+    source: new VectorSource({
+      wrapX: false,
+      features: new olCollection(),
+    }),
+    style: this.ngeoFeatureHelper_.getHighlightStyle(),
```

Comment:
I think that the style should be set via the `options` (through gmf-editfeature-options, and gmf-editfeatureselector-options).
But it looks like this component currently don't use the "options design" (search for `options` in `contribs/gmf/apps/desktop/index.html.ejs` and follow the logic for examples).

You can merge like this, and create a Jira ticket to move the style and the others components attributes to options OR you can set that directly in this PR. As you want.

(I'll be not very present today, but you can ask Sbrunner)